docs/metrics.md

```diff
@@ -0,0 +1,153 @@
+# Tab Split Metrics
+
+Metrics collections and analysis plan for the Tab Split experiment for [Firefox Test Pilot](https://testpilot.firefox.com).
+
+## Analysis
+
+Quantitative data collected in this experiment will be used to answer the following high-level questions:
+
+* How often do users split.
+* In any given session, what amount of active time is spent in a split?
+* Are splitters already heavy tab users?
+* Do splitters have wide screen devices?
+* What are common split ratios?
+* Do users understand the basic functionality of tab split
+  * Splitting and unsplitting
+  * Dragging
+  * Changing orientation
+  * Giving feedback
+
+## Collections
+
+### Custom Dimensions
+* `cd1` - orientation of split. One of `horizontal` or `vertical`.
+* `cd2` - width of browser window.
```

Comment:
- Specify viewport size
- Ensure we can bucket these sizes in the reports, or else we should bucket them before submitting